recipes/lyteidl/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "lyteidl" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lyteidl-{{ version }}.tar.gz
+  sha256: 9c5ebe96615e4fd03dd393c49f6356285be8e3860c58271ac7c8babc170165a8
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py>311]
```

Comment:
This should probably be a `noarch:python` recipe:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

The python version can be restricted and the selector on `typing` is unncessary since conda forge only supports `python>=3.9`.